appspec.yml

```diff
@@ -1,18 +0,0 @@
-version: 0.0
-os: linux
-files:
-  - source: /
-    destination: /app
-hooks:
-  AfterInstall:
-    - location: scripts/after_install.sh
-      timeout: 600
-      runas: root
-  ApplicationStart:
-    - location: scripts/application_start.sh
-      timeout: 600
-      runas: root
-  ApplicationStop:
-    - location: scripts/application_stop.sh
-      timeout: 60
-      runas: root
```

Comment:
I thought I looked, but when you say buildspec that sounds more like what I was looking for.  I'll go take another look to see what I find.

Reply:
I was going to investigate this, but after having so many problems with docker the last week I will just defer figuring out about this file to a later date since it's not necessary to proceed with the rest of the PR.

Reply:
Yeah no worries, punt it!